### ISSUE TITLE
Plausibility Warnings

### ISSUE THREAD

**Issue by mhchayan** [matsim-org/pt2matsim#94]
I am new to MATSim and also new to Java. I am trying to model Maryland, USA. I downloaded the osm file for Maryland and GTFS files for Maryland (21 agencies). As this package can take GTFS data from one zip file, I merged all the data and used it. But finally, it shows the following warnings. Can you please suggest if I can ignore these warnings or I have to change/update/format any files? If so, how to do that? Thanks in advance.

> Plausibility check for 5444 transit routes finished.
> 2884 	 artificial links
> 1073 	 loop warnings
> 911 	 direction change warnings
> 4845 	 travel time warnings
===============================

**Comment by polettif** [comment]
[I'll refer to the wiki entry](https://github.com/matsim-org/pt2matsim/wiki/Mapping-a-MATSim-schedule-to-a-MATSim-network#check-mapped-schedule-plausibility) first but I'll need to update it a bit. 

Basically, PlausibilityChecks only show you where there _might_ be problems with the network or schedule (thus warnings and not errors). It's more about having a look at them and less about eliminating them all. So in general I suggest inspecting the generated geojson files with [QGIS](https://qgis.org/) or [via](https://www.simunto.com/via/) to see whether where the warnings are located. For your case especially the travel time warnings are worth having a look at.

There are two ways to deal warnings: Check and then ignore if it is a false positive. Or you can adjust the network and run the PTMapper again, maybe with different config parameters. I'd refrain from changing the schedule because it's more difficult to do so and errors are more likely to be within the network. If your network is based on OSM, it's best to correct errors (like missing streets/bus lanes or wrong classified streets) on Openstreetmap, before converting to MATSim. It is also possible that you have to adjust some settings in the converter, e.g. for neighbourhood street etc. I wouldn't fix the MATSim network by hand (for example with JOSM), as it's not really  reproducible by code. Finally, there are a some settings/thresholds in the PlausibilityCheck class that you could adjust.

To summarize: There is no general approach to "fixing" those warnings and it depends on your use case whether this is even necessary.

**Comment by mhchayan** [comment]
Thank you for the suggestions!
